HW2/P2/P2.txt

```diff
@@ -0,0 +1,55 @@
+After testing, I found that serial execution was the fastest of the three. My guess is that the overhead of acquiring/releasing locks trumps the speedup gained from parallel execution.
+
+I didn't see much difference in execution time for correlated and uncorrelated. In fact, medium grained correlated had a slightly longer execution time. 
```

Comment:
This is surprising to me. My medium-grained code, with N~20 was the fastest for each case. However, OpenMP is pretty weird sometimes...it's hard to predict what it is going to do.